### PR TITLE
merging todoapp into env

### PR DESCRIPTION
**Pull request by MonaMarchetti**

### CODE REVIEW

File: ToDoProject/App.js [hunk 1]
```diff
@@ -1,19 +1,9 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React from "react";
+import { View, Text } from "react-native";
```
Comment (willmoffat) [inline ToDoProject/App.js]: Unused components

File: ToDoProject/src/Main.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+  save = async () => {
+    try {
+      AsyncStorage.setItem(TODO_STORAGE_KEY, JSON.stringify(this.state.todos));
+    } catch (e) {
+      console.log("Error while storing Todo Items >", e);
+    }
+  };
+
+  deleteKeys = async () => {
```
Comment (willmoffat) [inline ToDoProject/src/Main.js]: Consider renaming for clarity, how about `deleteAllTodos`

File: ToDoProject/src/Main.js [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    try {
+      const todos = await AsyncStorage.removeItem(TODO_STORAGE_KEY);
+      this.seState({
```
Comment (willmoffat) [inline ToDoProject/src/Main.js]: Why is this needed?

File: ToDoProject/src/Main.js [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+      return;
+    }
+    const todos = this.state.todos;
+    const todo = {
+      title: this.state.todo,
+      completed: false,
+      createdOn: Date.now(),
+      notes: "",
+      dueDate: null,
+      remindMe: false,
```
Comment (willmoffat) [inline ToDoProject/src/Main.js]: Looks like `notes, dueDate, remindMe` are not used. Consider deleting or commenting out.

File: ToDoProject/src/Main.js [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+    };
+    const addedTodos = todos.concat([todo]);
+    this.setState({ todos: addedTodos, todo: "" });
+    this.save();
+  };
+
+  checkBoxToggle = i => {
+    const todos = this.state.todos;
+    const checkedTodos = todos.map((todo, j) => {
+      if (j === i) {
```
Comment (willmoffat) [inline ToDoProject/src/Main.js]: nit: consider ` if (j !== i) { return todo }`

**Review by willmoffat — APPROVED** [review]
Looks good, just a few minor suggestions.

File: ToDoProject/App.js (hunk 1 above)
Comment (MonaMarchetti) [reply ToDoProject/App.js]: done

File: ToDoProject/src/Main.js (hunk 2 above)
Comment (MonaMarchetti) [reply ToDoProject/src/Main.js]: done

File: ToDoProject/src/Main.js (hunk 3 above)
Comment (MonaMarchetti) [reply ToDoProject/src/Main.js]: It's not... 
It's deleted now

File: ToDoProject/src/Main.js (hunk 4 above)
Comment (MonaMarchetti) [reply ToDoProject/src/Main.js]: done

File: ToDoProject/src/Main.js (hunk 5 above)
Comment (MonaMarchetti) [reply ToDoProject/src/Main.js]: done
